picard/coverart/__init__.py

```diff
@@ -95,7 +93,7 @@ def _coverart_downloaded(self, coverartimage, data, http, error):
         self.album._requests -= 1
 
         if error:
-            self.album.error_append('Coverart error: %s' % http.errorString())
+            self.album.error_append("Coverart error: %s" % http.errorString())
```

Comment:
Just because I don't understand: Why did this get changed to double quote, while in lines 128 following strings are changed from double quote to single quote?

Reply:
Well, my idea is that strings that are English phrases are likely to change to include a single quote at any moment, but that identifiers and keys are very unlikely to contain one.

Example:

```python
f("It is red")
f("It's red")
d['key']
```

Changing from `"It is"` to `"It's"` doesn't require any change regarding quotes, while changing from `'It is'` does.

At least, that the "consistency" I'm trying to apply since a while.

Reply:
Understood. Thanks for the explanation. Makes sense to me, I'll try to follow this scheme as well.

Reply:
I tried to clarify this: https://github.com/metabrainz/picard/pull/2322